Game/Source/GDKTestGyms/Private/UserExperienceComponent.cpp

```diff
@@ -32,10 +32,11 @@ void UUserExperienceComponent::StartRoundtrip()
 {
 	if (OpenRPCs.Contains(RequestKey))
 	{
+		UE_LOG(LogTemp, Warning, TEXT("UserExperience failure: Previous round trip didn't complete in time (%d)."), RequestKey);
```

Comment:
If this is a "failure", will we fail NFRs based on the presence of this log?

Reply:
No it won't, as the client logs aren't parsed. But this log would most likely indicate that the UX metric has failed, and having "UserExperience failure" is useful for bulk searching.